APE1.rst

```diff
@@ -145,14 +153,27 @@ An APE's status can
   rationales, so that new community members can understand why a decision was
   made.
 
-Modifications to an APE
-.......................
+* "Withdrawn": If the APE authors no longer feel that their proposal is feasible,
+  they can choose to withdraw the APE. A withdrawn APE is in effect equivalent
+  to a Rejected APE in that it will not be implemented.
+
+* "Abandoned": If an APE is present in the repository but has been labelled
+  as "Discussion" for at least a year without any sign of further discussion
```

Comment:
@aaryapatil thought a year might be too long but I'll let her comment further if that is indeed the case.

Also, should we unleash stale bot on this repo then? I think this will have to be a custom bot since we will be scouring merged documents, not open PRs.

Reply:
I think a year seems fine now that I've gone through it again.